tests/test_systeminfo.py

```diff
@@ -0,0 +1,85 @@
+from .reg_mock import RegistryMock, RegistryKeyMock, RegistryValueMock, LoggerMock
+from Registry.Registry import RegSZ, RegBin
+import pytest
+
+from regrippy.plugins.systeminfo import Plugin as plugin
+
+
+@pytest.fixture
+def mock_software():
+    key = RegistryKeyMock.build("Microsoft\\Windows NT\\CurrentVersion")
+    reg = RegistryMock("SOFTWARE", "software", key.root())
+
+    key_install_date = RegistryKeyMock("InstallDate", key)
+    key_owner = RegistryKeyMock("RegisteredOwner", key)
+    key_product_name = RegistryKeyMock("ProductName", key)
+
+    val1 = RegistryValueMock("InstallDate", 1523364100, RegBin)
+    val2 = RegistryValueMock("RegisteredOwner", "Some Dude", RegSZ)
+    val3 = RegistryValueMock("ProductName", "Windows 7 Professional", RegSZ)
+
+    key.add_child(key_install_date)
+    key.add_child(key_owner)
+    key.add_child(key_product_name)
+
+    key.add_value(val1)
+    key.add_value(val2)
+    key.add_value(val3)
+
+    return reg
+
+
+@pytest.fixture
+def mock_control():
+    host_name = RegistryKeyMock.build("ControlSet001\\Control\\ComputerName\\ComputerName")
+    reg = RegistryMock("SYSTEM", "system", host_name.root())
+    reg.set_ccs(1)
+
+    val1 = RegistryValueMock("ComputerName", "TestPC", RegSZ)
+    host_name.add_value(val1)
+
+    current_control_set = reg.open("ControlSet001\\Control")
+    current_control_set_window = RegistryKeyMock("Windows", current_control_set)
+    current_control_set.add_child(current_control_set_window)
+
+    val2 = RegistryValueMock("ShutdownTime", b'\xe40\xa5!\xed\xf7\xd3\x01', RegBin)
+    current_control_set_window .add_value(val2)
+    return reg
+
+
+@pytest.fixture
+def mock_services():
+    interface1 = RegistryKeyMock.build("ControlSet001\\Services\\Tcpip\\Parameters\\Interfaces\\{456A6A17D-21FC-123F-A689-A846D86EC008}")
+    reg = RegistryMock("SYSTEM", "system", interface1.root())
+    reg.set_ccs(1)
+
+    interface1.add_child('DhcpIPAddress')
```

Comment:
`RegistryKeyMock.add_child()` expects a `RegistryKeyMock` object, not an `str`. I believe you just forgot to remove this line :smile: 
```suggestion
```